### DISCUSSION TITLE
Prime esperienze nel domotizzare un impianto di riscaldamento

### DISCUSSION THREAD

**Discussion by marco-galli** [indomus/forum#2931] [GENERALE - Discussioni generiche]
Ciao tutti,

Avrei bisogno di consigli per domotizzare il riscaldamento di uno spazio gestito da un'associazione di cui faccio parte.

Lo spazio si compone di tre ambienti da circa 20-25mq ciascuno, piú un piccolo bagno.
Gli impianti sono già presenti e non è previsto effettuare lavori importanti.
L'impianto di riscaldamento è alimentato da una caldaia a condensazione che serve un impianto a pavimento, controllata da un unico termostato, alimentato a batterie stilo. Al termostato arrivano solo due fili elettrici, presumibilmente dalla caldaia.

I motivi per cui cerco di domotizzare il riscaldamento sono diversi:
- Lo spazio associativo non è sempre vissuto, capitano periodi in cui ci sono persone tutti i giorni, altri in cui non c'è nessuno per diversi giorni. Essendo a pavimento radiante, sarebbe utile poter accendere il riscaldamento con un certo anticipo (da qualche ora prima ad anche un giorno prima se fa particolarmente freddo) rispetto al momento in cui arrivano gli associati.
- Programmare con piú granularità rispetto al semplice ciclo giornaliero.

I desiderata sono:
- Controllo del termostato sia localmente che da remoto (app mobile o da pc).
- Programmazione non solo giornaliera ma settimanale o, se possibile, anche più granulare.
- Possibilità di bloccare l'interazione con il termostato a muro. Essendo un luogo 'pubblico', in modo da impedire che qualche genio si metta a smanettare incasinando le impostazioni.
- Se fosse interfacciabile con un termometro remoto (piazzato fuori dall'edificio) sarebbe ottimo, in modo da poter almeno registrare la temp. esterna, se non addirittura programmare i tempi di anticipo sull'accensione del termostato stesso in base alla temperatura esterna (ma questo forse è possibile solo nella mia fantasia).

Nei locali in questione c'è una connessione internet con router wifi.
Non credo valga la pena di mettere in piedi un hub personale (Home Assistant ad esempio) soltanto per un termostato, anche se mi piacerebbe non dover passare attraverso un cloud del produttore. Ma ditemi voi se sia indicato. Nel caso, ho un po di competenze col mondo linux.

Mi affaccio per la prima volta al mondo della domotica emi piacerebbe avere le vostre opinioni e magari dei consigli su qualche modello/prodotto idoneo, o magari anche solo la tipologia di prodotto che può fare al mio caso.

Saluti
Marco

**Comment by indomus** [comment]
per capire "dove sei" in termini culturali: [questo](https://indomus.it/nozioni-di-base/) lo hai già letto tutto?

**Reply by marco-galli** [reply]
Sisi, ho letto tutto quello che c'è sotto il menu "Nozioni di base" e "Hub personali", Ho letto anche qualcosa sulla sezione Software e su quella Hardware.
Ho delle buone skill in programmazione e capisco qualcosina di reti. Purtroppo la parte che mi manca molto è quella di elettrotecnica.
Anzi, grazie a tutto il team di indomus.it per tutti i contenuti che mette a disposizione del vasto pubblico.

**Reply by indomus** [reply]
grazie a te.
usare un HUB ha i suoi enormi vantaggi, chiaro è che se devi configurare solo il riscaldamento, ti basta scegliere una delle tante soluzioni autonome che ci sono sul mercato (tado, netatmo, meross per citarne alcune da noi recensite) e puoi andare dritto. il problema è il controllo in rete locale, cosa che non puoi fare: sono tutte soluzioni basate su cloud, l'alternativa è adottare Home Assistant o un HUB personale a scelta. 
solitamente i termostati hanno il blocco bambini, quindi puoi inibire le modifiche effettuate fisicamente sul controller. sul tema del sensore esterno, soluzioni come tado utilizzano al geolocalizzazione dell'impianto per fare precisamente quel che richiedi, in modo del tutto automatico.

**Comment by Bergoglio** [comment]
Valuta bene perché l'appetito vien mangiando : oggi inizi con il termostato, domani aggiungi il condizionatore, dopo domani l'illuminazione..... e via discorrendo.
Home Assistant é potente, ma bisogna lavorarci sopra; ti permetterebbe tutto quello che chiedi, ma bisogna svilupparlo : considera che è il motore, a te il compito di crearci sopra il mezzo.

In merito alle questioni generali del riscaldamento : attenzione, in Italia (o meglio, in italietta) abbiamo il vizio di impostare molti cambi di ciclo nella giornata (2 ore temp alta, 3 ore spento poi 1 ora temp alta poi di nuovo in ridotto etc.) ma non va bene perché 
1) hai un riscaldamento a pavimento che presenta una forte inerzia, per fare cicli brevi dovresti cuocere i piedi degli associati oltre ad andare contro la normativa vigente che ti impone una temperatura massima alla piastrella di 29 gradi (per me già duri da sopportare)
2) comunque a fare cicli brevi consumi di più perché devi far viaggiare la caldaia tirata per il collo ad alta temperatura, esattamente dove il recupero dalla condensazione è nullo e quindi lavori in fasce a rendimento più basso (94 - 96% invece che 104 - 108%)

Curiosità : in cosa sai programmare ?
